twister: for twister can run tests on ITE board.

fuhaolei opened the pull request:
Set board flasher, so twister can run tests on ITE(it8xxx2_evb) board.
Add supported drivers of it8xxx2_evb board, so twister could run driver
tests without skipping.

Signed-off-by: rainer754 <haolei.fu@intel.com>

enjiamai commented:
Please fix the compliance check, thx.

tejlmand reviewed (dismissed):
lgtm

nashif commented:
> Signed-off-by: rainer754 [haolei.fu@intel.com](mailto:haolei.fu@intel.com)

you need to fix that.

tejlmand reviewed (approved):
reapproved

fuhaolei committed:
twister: for twister can run tests on ITE board.

Set board flasher, so twister can run tests on ITE(it8xxx2_evb) board.
Add supported drivers of it8xxx2_evb board, so twister could run these
driver tests without skipping.

Signed-off-by: Fu Haolei <haolei.fu@intel.com>
